examples/47-pixelformats/pixelformats.cpp

```diff
@@ -256,6 +303,8 @@ class ExamplePixelFormats : public entry::AppI
 	uint32_t m_height;
 	uint32_t m_debug;
 	uint32_t m_reset;
+	float    m_previewSize = 50.0f;
```

Comment:
Any reason why this isn't a global const?

Reply:
`m_previewSize` is modifiable with slider, so user can increase preview size.

![image](https://user-images.githubusercontent.com/1197433/185976320-83c2cc1e-fa60-4ee0-b3ab-6ea045887bbe.png)

Reply:
Oh I see!